scale-demo/nginx-rc.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    run: nginx
```

Comment:
run doesn't allow you to set a custom request CPU.  I could use --override, I suppose, but I think that's uglier.